Core/Constants/IdentityRoles.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Core.Constants
+{
+    public enum IdentityRoleNames
```

Comment:
Why don't we create string constants instead of enum so that we don't convert them to a string later?